docs/book/src/user/quick-start.md

```diff
@@ -587,7 +587,7 @@ For the purpose of this tutorial, we'll name our cluster capi-quickstart.
 
 ```bash
 clusterctl generate cluster capi-quickstart \
-  --kubernetes-version v1.19.7 \
+  --kubernetes-version v1.19.11 \
```

Comment:
Upgraded to the patch version we're testing in CI

Reply:
We can probably also install 1.21

Reply:
I hope so :). But I think it would be good to test the same version in our "quick start" e2e test we document here. 

So if we want to switch to 1.21 I think it would be better to also change the e2e test (and do both in a separate PR).

/cc @fabriziopandini

Reply:
I'm +1 to jump to a newer releases in the quick-start and in the e2e; ideally we should stay in lockstep with K8s.
If we are going for 1.21 let's jump to the .2 path release, because it contains the fix for CoreDNS image and for etcd ephemeral storage.

Reply:
Good, I will create a PR for it